ajax/libs/bodymovin/package.json

```diff
@@ -0,0 +1,33 @@
+{
+  "name": "bodymovin",
+  "version": "4.5.3",
+  "description": "After Effects plugin for exporting animations to SVG + JavaScript or canvas + JavaScript",
+  "repository": {
+    "type": "git",
+    "url": "git+https://github.com/bodymovin/bodymovin.git"
+  },
+  "keywords": [
+    "animation",
+    "canvas",
+    "svg",
+    "after effects",
+    "plugin",
+    "export"
+  ],
+  "license": "MIT",
+  "homepage": "https://github.com/bodymovin/bodymovin/wiki",
+  "author": {
+    "name": "airnan",
+    "email": "hernantorrisi@gmail.com"
+  },
+  "filename": "bodymovin.min.js",
+  "npmName": "bodymovin",
+  "npmFileMap": [
+    {
+      "basePath": "build/play",
+      "files": [
+        "**/bodymovin.*"
```

Comment:
The front-end files are at "build/player" instead of "build/play". 
Is that the error?